lib/Backend/Commands.hs

```diff
@@ -316,20 +289,13 @@ buildCopyOperations oldPath newPath force = do
       Left entry -> copyEntry entry
       Right dir -> copyDir dir
 
-  -- Checks that the paths we're trying to copy entries to aren't already occupied
-  -- by an existing directory.
-  checkCopyOperations operations
-    do \(CopyOperation _ new) -> pathIsDirectory (new ^. E.path)
-    do CPRDestinationIsDirectory . fmap getOperationPaths
-
-  when (not force) do
-    -- Checks that the paths we're trying to copy entries to aren't already occupied
-    -- by an existing entry.
-    checkCopyOperations operations
-      do \(CopyOperation _ new) -> pathIsEntry (new ^. E.path)
-      do CPREntryAlreadyExists . fmap getOperationPaths
+  sequenceA <$> forM operations validateCopyOperation >>= \case
+    Failure createErrors -> throw $ CPRCreateErrors createErrors
+    Success _ -> pure operations
 
-  pure operations
+  -- sequenceA <$> forM (operations <&> coNew) (checkCreateEntry force) >>= \case
+  --   Failure createErrors -> throw $ CPRCreateErrors createErrors
+  --   Success _ -> pure operations
```

Comment:
Forgot to delete this commented out code.

Reply:
Done